### ISSUE TITLE
Is PEAR supposed to be required?

### ISSUE THREAD

**Issue by mikehaas763** [Pawka/phrozn#78]
Following the websites "Installation" page, I was getting an exception when trying to run phr relating to a file not being found. I figured out that when I installed PEAR (sudo apt-get install php-pear) it would then work. Aren't the PEAR dependencies supposed to be included in the phrozn package? If PEAR is a dependency, it should be included on the installation page.

**Comment by farazdagi** [comment]
This should not be an issue once #74 hits the master branch (should be next week when I plan to go through some due pull requests and issues).

**Comment by kenjis** [comment]
0.5.4 still requires PEAR.

> PHP Warning:  require_once(PEAR/Exception.php): failed to open stream: No such file or directory in .../phrozn/vendor/pear/console_commandline/Console/CommandLine/Exception.php on line 28

**Comment by haruair** [comment]
Is it fixed? I got a same warning in 0.5.4.

```
Warning: require_once(PEAR/Exception.php): failed to open stream: No such file or directory in /Users/dev/phrozn/vendor/pear/console_commandline/Console/CommandLine/Exception.php on line 28
```

**Comment by mikehaas763** [comment]
Not that I'm saying console_commandline is inferior but I think it would be pretty neat to see a PR come in replacing it with Symfony/Console. Even though I haven't created a static site since about oh 8 months ago.

**Comment by Pawka** [comment]
It was temporary fixed on #101. If anybody wants get rid all PEAR stuff, you are welcome :-)
